ember-responsive-image@5.0.0-beta.0 / Cannot find module

After installing the beta i get this error when running ember serve:

Cannot find module '/ < ember-project-dir > /config/addons'
Require stack:
/ < ember-project-dir > /node_modules/.pnpm/ember-addon-config@0.1.0_@glint+template@1.2.1/node_modules/ember-addon-config/index.js

Its a pretty new ember app with embroider.

Thanks for your work on this addon.

Greetings
Benjamin

--- a/.changeset/thick-eagles-collect.md
+++ b/.changeset/thick-eagles-collect.md
@@ -1,0 +1,5 @@
+---
+"ember-responsive-image": patch
+---
+
+Fix broken build when no addon config is found

--- a/packages/ember-responsive-image/package.json
+++ b/packages/ember-responsive-image/package.json
@@ -36,7 +36,7 @@
     "@embroider/macros": "^1.0.0",
     "blurhash": "^2.0.0",
     "decorator-transforms": "^1.0.2",
-    "ember-addon-config": "^0.1.0",
+    "ember-addon-config": "^0.1.1",
     "ember-style-modifier": "^4.0.0"
   },
   "peerDependencies": {

